addons/better-img-uploads/userscript.js

```diff
@@ -110,8 +110,8 @@ export default async function ({ addon, console, msg }) {
     let processed = new Array();
 
     for (let file of files) {
-      if (file.type.includes("svg")) {
-        //The file is already a svg, we should not change it...
+      if (file.type.endsWith("/svg") || file.name.endsWith(".sprite3")) {
```

Comment:
```suggestion
      if (file.type.endsWith("/svg+xml") || file.name.endsWith(".sprite3")) {
```

Reply:
Yeah, let's be careful with the MIME types here.